src/mlpack/methods/ann/activation_functions/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(SOURCES
   softplus_function.hpp
   swish_function.hpp
   mish_function.hpp
+  isrlu_function.hpp
```

Comment:
This change looks unrelated; can you make sure you are developing your work for each PR on a separate branch, so that the changes don't get mixed up?  Thanks. :+1:

Reply:
I'm really sorry about this. I'll make seperate branches from now on.